contrib/nydus-backend-proxy/README.md

```diff
@@ -0,0 +1,104 @@
+# nydus-backend-proxy
+A http blob server based on Rocket to serve static file for nydusd.
+
+In some scenarios such as [sealer](https://github.com/alibaba/sealer), there is no registry or OSS available. So we need a simple http server instead of deploying a registry server to serve blob for nydusd, in order to boost up cluster image distribution. This blob server support HEAD and GET request from nydusd to check and fetch blob.
+
```

Comment:
A simple HTTP server to serve a local directory as blob backend for nydusd.

In some scenarios such as [sealer](https://github.com/alibaba/sealer), it uses nydus to boost up cluster image distribution. There is no registry (OCI distribution) or OSS service available for blob storage, so we need a simple HTTP server to serve a local directory as blob backend for nydusd. This server exposes OCI distribution like API to handle HTTP HEAD and range GET requests from nydusd for checking and fetching blob.